js_modules/dagster-ui/packages/ui-core/src/asset-graph/useAssetGraphExplorerFilters.oss.tsx

```diff
@@ -206,33 +206,44 @@ export function useAssetGraphExplorerFilters({
   filters.push(tagsFilter);
   filters.push(ownerFilter);
   const {button, activeFiltersJsx} = useFilters({filters});
-  if (!filters.length) {
-    return {button: null, activeFiltersJsx: null};
-  }
+
+  const renderFilterBar = useCallback(
```

Comment:
Not too happy about exposing composability via this prop, but it seems simpler than yet another injected component. I can go with the InjectedComponent route though if we prefer that.